hoomd_organics/base/simulation.py

```diff
@@ -587,7 +838,23 @@ def run_NVT(
         self.operations.updaters.remove(std_out_logger_printer)
 
     def run_NVE(self, n_steps, write_at_start=True):
-        """"""
+        """
+        Run the simulation in NVE dynamics.
```

Comment:
I think it would be more correct to say "Run the simulation in the NVE ensemble". The dynamics thing really only applies to the Langevin method